lib/services/google_analytics.rb

```diff
@@ -0,0 +1,63 @@
+require 'google/apis/analyticsreporting_v4'
+require 'googleauth'
+
+module Services
+  class GoogleAnalytics
+    include Google::Apis::AnalyticsreportingV4
+    include Google::Auth
+
+    def client(scope: "https://www.googleapis.com/auth/analytics.readonly")
+      @client ||= AnalyticsReportingService.new
+      @client.authorization ||= ServiceAccountCredentials.make_creds(
+        json_key_io: File.open(ENV["GOOGLE_AUTH_CREDENTIALS"]),
```

Comment:
I wonder if there is another way to load the credentials. Something like:

```ruby
{
  "type": "service_account",
  "project_id": ENV['PROJECT_ID'],
  "private_key_id": ENV['PRIVATE_KEY_ID'],
  "private_key": ENV['PRIVATE_KEY'],
  "client_email": ENV['CLIENT_EMAIL'],
  "client_id": ENV['CLIENT_ID'],
  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
  "token_uri": "https://accounts.google.com/o/oauth2/token",
  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/govuk-content-performance-mngr%40govuk-content-performance-mngr.iam.gserviceaccount.com"
}
```

It seems like an easier way to configure parameters based on environment variables.